test/fixtures/user-integration-app/server/server.js

```diff
@@ -0,0 +1,14 @@
+var loopback = require('../../../../index');
+var boot = require('loopback-boot');
+var path = require('path');
+var app = module.exports = loopback();
+app.enableAuth();
+boot(app, __dirname);
+app.use(loopback.favicon());
+app.use(loopback.cookieParser({secret: app.get('cookieSecret')}));
+app.use(loopback.token({model: app.models.AccessToken}));
+var apiPath = '/api';
+app.use(apiPath, loopback.rest());
+app.use(loopback.static(path.join(__dirname, 'public')));
+app.use(loopback.urlNotFound());
+app.use(loopback.errorHandler());
```

Comment:
Is there any particular reason why you are not using `middleware.json` and middleware phases in this app?

Are your tests relying on `cookieParser` at all? And why do they need static files from the `public` directory?